Makefile

```diff
@@ -23,5 +23,4 @@ deploy: build deploy-setup
 	rm main.py
 	rm requirements.txt
 format:
-	black . -l 79
```

Comment:
can we leave this? we don't have much python but some

Reply:
Sure!